charts/enterprise/vaultwarden/questions.yaml

```diff
@@ -203,11 +203,18 @@ questions:
                     - variable: ssl
                       label: "Enable SSL connection"
                       schema:
-                        type: boolean
-                        default: true
+                        type: string
+                        default: "starttls"
+                        enum:
+                          - value: "starttls"
+                            description: "STARTTLS (587)"
+                          - value: "force_tls"
+                            description: "FORCE_TLS (465)"
+                          - value: "off"
+                            description: "OFF (25)"
```

Comment:
I'm afraid this change will break scale UI. due to changing type
Better to use a new variable name.

Reply:
Makes sense I will change it to tls

Reply:
changed to `tls`